joomla/joomla-cms
# [associations] Plugin link in message

### Summary of Changes
If you go to the Redirects or Smart Search component without the required plugin enabled you will get a message to enable it and the message includes a direct link to the plugin itself. 
<img width="803" alt="screenshotr21-38-40" src="https://cloud.githubusercontent.com/assets/1296369/23972591/4f8053de-09ca-11e7-8870-756eaeb42f55.png">

The new Multilingual Associations component had a similar message but there is no link to the plugin

<img width="835" alt="screenshotr21-39-09" src="https://cloud.githubusercontent.com/assets/1296369/23972621/686b025e-09ca-11e7-95d8-d5ddd59e0c34.png">

After applying this PR the message will change to have a link to the plugin in the message itself. 

<img width="572" alt="screenshotr21-58-35" src="https://cloud.githubusercontent.com/assets/1296369/23972669/8fb30708-09ca-11e7-8134-7a8d39a3c2e0.png">





### Testing Instructions
Make sure that  Items Associations is not enabled in the Language Filter plugin.
Open the Multilingual Associations component
Before the PR the message is just text
After the PR the message includes a link to the Language Filter plugin


### Documentation Changes Required
None
